src/raven.js

```diff
@@ -652,9 +650,9 @@ Raven.prototype = {
                 target = '<unknown>';
             }
 
-            self.captureBreadcrumb('ui_event', {
-                type: evtName,
-                target: target
+            self.captureBreadcrumb({
+                category: 'ui.' + evtName, // e.g. ui.click, ui.input
```

Comment:
@dcramer gave me impression that `type` should strictly be for schema. (Makes me wonder why we don't just call it `schema`.) Since there is no `data` to enforce here ... not sure what to do.